fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -19,6 +22,7 @@ import com.microsoft.fluentui.tokenized.tabItem.TabItem
 
 data class TabData(
     var title: String,
+    var talkbackDescription: String? = null,  //Custom announcement for Talkback
```

Comment:
rename this to something generic as this will be used by all the screen readers and not specifically talkback

Reply:
Updating to 'accessibilityDescription'